deny.toml

```diff
@@ -1,3 +1,6 @@
+# Based on https://github.com/EmbarkStudios/cargo-deny/blob/0.16.0/deny.template.toml
+# (diff with that version to see what has been manually changed here)
```

Comment:
I took everything from that template, except for the manual changes mentioned by my other GitHub comments.